news/2019-02-04-community-choice-2018-results.md

```diff
@@ -0,0 +1,147 @@
+### osu!taiko
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/xinobikill.jpg)](https://osu.ppy.sh/beatmapsets/742966)
+
+**121** drum fanatics stepped forward and said "hell yes" to **[SOUND HOLIC feat. Nana Takahashi - XINOBIKILL](https://osu.ppy.sh/beatmapsets/742966)** by [Skull Kid](https://osu.ppy.sh/users/3044264), in some of the closest voting we've ever observed in a Community Choice poll *or* a "Best Of" to date.
+
+Seriously, at some points, there were less than 3 votes splitting first, second, and third place. When we tweeted about the votes drawing to a close, there was a two-way tie for first place for at least a day before the votes were slated to close.
+
+Crazy stuff. Congratulations!
+
+The finalists were:
+
+* [Porno Graffiti - The Day feat. Aruvn (dj-Jo Remix) Extended Version by Chromoxx](https://osu.ppy.sh/beatmapsets/693174) with **104** votes total.
+
+* [t+pazolite - Ultrasonic Anthem (Kobaryo's FTN-Remix) by tasuke912](https://osu.ppy.sh/beatmapsets/646093) with **94** votes total.
+
+* [Unknown Artist - Miniministop Hitoyasumi no Uta\[Mokuyoubi\]Gakusei no Uta by Chromoxx](https://osu.ppy.sh/beatmapsets/705423) with **74** votes total.
+
+* [Vickeblanka - Black Rover (TV Size) by Sotarks](https://osu.ppy.sh/beatmapsets/781509) with **72** votes total.
+
+* [ESTi - HELIX (Edit ver.) by hikikochan](https://osu.ppy.sh/beatmapsets/625729) with **71** votes total.
+
+* [ClariS - SHIORI vs. Hitorigoto by Sotarks](https://osu.ppy.sh/beatmapsets/707380) with **67** votes total.
+
+* [XXXTENTACION - Jocelyn Flores by Chromoxx](https://osu.ppy.sh/beatmapsets/705133) with **66** votes total.
+
+* [Will Stetson - Harumachi Clover (Swing Arrangement) [Dictate Edit] by Sotarks](https://osu.ppy.sh/beatmapsets/842412) with **63** votes total.
+
+* [KASAI HARCORES - Cycle Hit by Axer](https://osu.ppy.sh/beatmapsets/668239) with **63** votes total.
+
+[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/75)
+
+### osu!catch
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/asphyxia.jpg)](https://osu.ppy.sh/beatmapsets/758101)
+
+With **182** votes, the collaborative sets by [Ascendance](https://osu.ppy.sh/users/2931883), [Spectator](https://osu.ppy.sh/users/702598) and [Hareimu](https://osu.ppy.sh/users/4138746) on **[Co shu Nie - asphyxia (TV edit)](https://osu.ppy.sh/beatmapsets/758101)** hosted by [Monstrata](https://osu.ppy.sh/users/2706438) strangled out the competition solidly with a nearly 50 point lead.
+
+Similar to osu!taiko, the voting for osu!catch was a super close endeavour all up until the last three or four days of voting, when asphyxia surged forward with a lead that the other sets could just not recover from.
+
+The agonizing choice many osu!catch players were forced to make is super obvious in the sheer vote gap between many of the finalist entries, shown purely in the fact that 2nd and 3rd place are literally one vote apart. 
```

Comment:
"were" literally one vote apart? Past tense consistency in this sentence probably